src/01-general/history.apib

```diff
@@ -24,6 +24,37 @@ Retrieve a list of records about an item.
                 + title: `Marc was on a call with John from this company` (string)
                 + description: `An extended description` (string)
                 + recorded_at: `2017-05-09T11:25:11+00:00` (string)
+                + type (enum[string])
+                    + Members
+                        + call
+                        + client_form_filled_out
+                        + client_satisation_mail_sent_project
+                        + client_satisation_mail_sent_ticket
+                        + company
+                        + contact
+                        + creditnote
+                        + custom
+                        + deal
+                        + edit
+                        + email
+                        + event
+                        + external_interaction
+                        + file
+                        + gmail_email
+                        + invoice
+                        + linkedin
+                        + mail_sent
+                        + mailing
+                        + meeting
+                        + note
+                        + order
+                        + permanent_note
+                        + product
+                        + project
+                        + sale
+                        + ticket_closed
+                        + timetracking
+                        + todo
```

Comment:
This is the list of types that exist today, which obviously makes no sense in the API. Stijn proposes a different list, but we'd have to agree on it. Not sure what to do.